src/unit_tests/type/test_type.cpp

```diff
@@ -215,7 +215,7 @@ TEST_CASE("Test rttr::type - MultipleClassInheritance", "[type]")
         CHECK(rttr_cast<FinalClass*>(&baseMultiC) != nullptr);
         CHECK(rttr_cast<FinalClass*>(&baseMultiD) != nullptr);
         CHECK(rttr_cast<FinalClass*>(&baseMultiE) != nullptr);
-
+        
```

Comment:
Not sure how picky the rttr maintainers are but often extraneous spaces are considered bad form.